db/deploy/tables/applications.sql

```diff
@@ -5,10 +5,55 @@ begin;
 
 create table if not exists connectivity_intake_public.applications (
   id serial not null,
+  owner uuid not null,
```

Comment:
we probably also want a trigger on create that sets the `NEW.owner = (select sub from connectivity_intake_private.session())` and an update trigger that sets `NEW.owner = OLD.owner`